<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunForm</class>
 <widget class="QDialog" name="RunForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="_runFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="_startTime"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="_toleranceLbl">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="_absToleranceLbl">
        <property name="text">
         <string>AbsTolerance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_startTimeLbl">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="_outputLbl">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="_descriptionLbl">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="_tolerance"/>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="_output"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_solverLbl">
        <property name="text">
         <string>Solver</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="_absTolerance"/>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QTextEdit" name="_description"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="_solver">
        <item>
         <property name="text">
          <string>QSS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CQSS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIQSS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIQSS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIQSS_BDF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSS3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIQSS3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSS4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DASSL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DOPRI5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CVODE_BDF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CVODE_AM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IDA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mLIQSS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mLIQSS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mLIQSS3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="_stopTimeLbl">
        <property name="text">
         <string>Stop Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="_stopTime"/>
      </item>
      <item row="7" column="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QCheckBox" name="_showAll">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>216</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Show all settings</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="_debugChk">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>216</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="_extendedFrame">
     <property name="minimumSize">
      <size>
       <width>642</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="2">
       <widget class="QLabel" name="_lpsLbl">
        <property name="text">
         <string>Logical Processes </string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="_debugGraphLbl">
        <property name="text">
         <string>Debug Graph</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="_scheduler">
        <item>
         <property name="text">
          <string>BINARY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RANDOM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LINEAR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="_period">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLineEdit" name="_metisSettings">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="_symDiff">
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="_parallelLbl">
        <property name="text">
         <string>Parallel</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="_debugGraphCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="_patohSettingsLbl">
        <property name="text">
         <string>Patoh Settings</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="_reorderPartitionCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="_parallel">
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QLineEdit" name="_imbalance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.01</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="_BDFMaxStepLbl">
        <property name="text">
         <string>BDF Max Step Size:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="_dtSynch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>ADAPTIVE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FIXED</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="_zcHystLbl">
        <property name="text">
         <string>Zero Crossing Hysteresis</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="_derDeltaLbl">
        <property name="text">
         <string>Derivative Delta</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="_dtSynchLbl">
        <property name="text">
         <string>Dt Synchronization </string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="_imbalanceLbl">
        <property name="text">
         <string>Imbalance</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="_metisSettingsLbl">
        <property name="text">
         <string>Metis Settings</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="_BDFPartLbl">
        <property name="text">
         <string>BDF Partition</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_symDiffLbl">
        <property name="text">
         <string>Symbolic Differentiation</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="_scotchSettingsLbl">
        <property name="text">
         <string>Scotch Settings</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="_bdfPartitionDepthlbl">
        <property name="text">
         <string>BDF Partition Depth</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="_commIntervalLbl">
        <property name="text">
         <string>Output Type</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="_BDFMaxStep"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="_commInterval">
        <item>
         <property name="text">
          <string>STEP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SAMPLED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DENSE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="_semiStaticLbl">
        <property name="text">
         <string>Semi-static Partitioning</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="_minStep"/>
      </item>
      <item row="6" column="3">
       <widget class="QComboBox" name="_partitionMethod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>METIS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HMETIS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SCOTCH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PATOH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MANUAL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QComboBox" name="_generateArchCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QSpinBox" name="_BDFPartitionDepth">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="_scotchSettings">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QCheckBox" name="_semiStaticChk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="_patohSettings">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="_lps">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="_periodLbl">
        <property name="text">
         <string>Period</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="_reorderPartitionLbl">
        <property name="text">
         <string>Reorder Partition</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Jacobian (CVODE/IDA)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="_minStepLbl">
        <property name="text">
         <string>Minimum Step Size</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="_dt">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QLineEdit" name="_BDFPart"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="_derDelta"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="_zcHyst"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="_pmLbl">
        <property name="text">
         <string>Partition Method</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="_jacobian">
        <item>
         <property name="text">
          <string>Sparse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_schedulerLbl">
        <property name="text">
         <string>Scheduler</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="_generateArchLbl">
        <property name="text">
         <string>Generate Arch</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="_dtLbl">
        <property name="text">
         <string>Dt tolerance</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QCheckBox" name="_test_methods_cbx">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="_test_methods_lbl">
        <property name="text">
         <string>QSS Test Methods:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="_runButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_startTime</tabstop>
  <tabstop>_stopTime</tabstop>
  <tabstop>_tolerance</tabstop>
  <tabstop>_absTolerance</tabstop>
  <tabstop>_solver</tabstop>
  <tabstop>_output</tabstop>
  <tabstop>_description</tabstop>
  <tabstop>_showAll</tabstop>
  <tabstop>_debugChk</tabstop>
  <tabstop>_commInterval</tabstop>
  <tabstop>_period</tabstop>
  <tabstop>_symDiff</tabstop>
  <tabstop>_derDelta</tabstop>
  <tabstop>_minStep</tabstop>
  <tabstop>_parallel</tabstop>
  <tabstop>_zcHyst</tabstop>
  <tabstop>_lps</tabstop>
  <tabstop>_scheduler</tabstop>
  <tabstop>_partitionMethod</tabstop>
  <tabstop>_dtSynch</tabstop>
  <tabstop>_dt</tabstop>
  <tabstop>_scotchSettings</tabstop>
  <tabstop>_metisSettings</tabstop>
  <tabstop>_patohSettings</tabstop>
  <tabstop>_semiStaticChk</tabstop>
  <tabstop>_jacobian</tabstop>
  <tabstop>_imbalance</tabstop>
  <tabstop>_reorderPartitionCbx</tabstop>
  <tabstop>_generateArchCbx</tabstop>
  <tabstop>_debugGraphCbx</tabstop>
  <tabstop>_BDFPart</tabstop>
  <tabstop>_BDFMaxStep</tabstop>
  <tabstop>_BDFPartitionDepth</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_runButtons</sender>
   <signal>accepted()</signal>
   <receiver>RunForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_runButtons</sender>
   <signal>rejected()</signal>
   <receiver>RunForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
